Sources/AsyncMoya/AsyncRequestWrapper.swift

```diff
@@ -0,0 +1,26 @@
+import Foundation
+import Moya
+
+#if swift(>=5.5)
+
+@available(OSX 10.15, iOS 13.0, tvOS 13.0, watchOS 6.0, *)
+internal class AsyncMoyaRequestWrapper {
+    
+    var performRequest: (CheckedContinuation<Result<Response, MoyaError>, Never>) -> Moya.Cancellable?
```

Comment:
We could consider creating a typealias for this completion handler / continuation so we don't have to copy it in the few places here, but this works too :-)
```suggestion
    typealias MoyaContinuation = (CheckedContinuation<Result<Response, MoyaError>, Never>)
    var performRequest: (MoyaContinuation) -> Moya.Cancellable?
```